acl-lib/src/cmr/acl/core.clj

```diff
@@ -154,6 +162,11 @@
   []
   (mem-cache/create-in-memory-cache :ttl {} {:ttl TOKEN_IMP_CACHE_TIME}))
 
+(defn create-access-constraints-cache
+  "Creates a cache for access constraint mapping."
+  []
+  (mem-cache/create-in-memory-cache :ttl {} {:ttl CONCEPT_MAP_CACHE_TIME}))
```

Comment:
Since this is an in-memory cache, are they are concerns with multiple requests coming in to different instances of access-control where only some have cached data?